hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+        },
+        { bits: "22:20",
+          name: "CSNTRAIL"
+          desc: '''CS_N Trailing Time.  Indicates the number of half sck cycles,
+                   CSNTRAIL+1, to leave between last edge of sck and the rising
+                   edge of cs_n. Setting this register to zero corresponds
+                   to the minimum delay of one-half sck cycle.'''
+          resval: 0
+        },
+        { bits: "19:17",
+          name: "CSNIDLE"
+          desc: '''Minimum idle time between commands. Indicates the minimum
+                   number of sck half-cycles to hold cs_n high between commands.
+                   Setting this register to zero creates a minimally-wide CS_N-high
+                   pulse of one-half sck cycle.'''
+          resval: 0
+        },
+        { bits: "15:0",
+          name: "CLKDIV"
+          desc: '''Core clock divider.  Slows down subsequent SPI transactions by a
+                   factor of (CLKDIV+1) relative to the core clock frequency.  The
+                   period of sck, T(sck), then becomes 2*(CLK_DIV+1)*T(core)'''
+          resval: 0
+        },
+      ]
+    },
+    { name: "COMMAND",
+      desc: "Command Register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "3:0",
+          name: "TX1_CNT",
+          desc: '''The number of 1-byte bursts to transmit in this command,
+                   at the single-line data width (8 sck clocks per cycle).
+                   All other data lines are high-Z during this command phase.
+                   This field is ignored when SPEED = 0.
+                   ''',
+          resval: "0x0"
+        },
+        { bits: "12:4",
+          name: "TXN_CNT",
+          desc: '''The number of 1-byte bursts to transmit in this command,
+                   at the full-data width (Determined by the SPEED field).
+                   ''',
+          resval: "0x0"
+        },
+        { bits: "16:13",
+          name: "DUMMY_CNT",
+          desc: '''Controls the number of sck dummy cycles (NOT bytes)
+                   inserted between end of the last TX byte and the beginning of
+                   the first RX byte.   No data is transmitted or received
+                   and all outputs are high-Z during this command phase.  This
+                   field is ignored when SPEED = 0''',
+          resval: "0x0"
+        },
+        { bits: "25:17",
+          name: "RX_CNT",
+          desc: '''In Dual or Quad mode, this is the number of 1-byte bursts
+                   to receive in this command.  This field is ignored when
+                   SPEED = 0, because the number of bytes received is always
+                   equal to the number of bytes transmitted in Standard SPI
+                   mode.'''
+          resval: "0x0"
+        },
+        { bits: "28:26",
+          name: "CSID",
+          desc: '''The Chip select ID of the device to receive the following
+                   command.''',
```

Comment:
Does this really mean "following", rather than "this"? It seems odd that some of the fields apply to the current command and some apply to the next one?

Reply:
Good point.  Fixed.